test/format-test.cc

```diff
@@ -2491,18 +2491,27 @@ TEST(FormatTest, FmtStringInTemplate) {
 
 #endif  // FMT_USE_CONSTEXPR
 
+// C++20 feature test, since r346892 Clang considers char8_t a fundamental
+// type in this mode. If this is the case __cpp_char8_t will be defined.
+#ifndef __cpp_char8_t
+// A UTF-8 code unit type.
+#define CHAR8_T fmt::char8_t
```

Comment:
I suggest replacing a macro with a using declaration.

Reply:
Done.

With c++11 using, it looks way way better now. Thank you.